srcpkgs/kcachegrind/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'kcachegrind'
 pkgname=kcachegrind
-version=18.04.2
+version=18.04.3
 revision=1
 build_style=cmake
 configure_args="-Wno-dev"
```

Comment:
is this still revelant ? it doesn't even look like a configuration switch for cmake

Reply:
Yes, that [cmake argument](https://cmake.org/cmake/help/latest/manual/cmake.1.html) can be safely dropped.